scripts/Muon/GUI/Common/results_tab_widget/results_tab_presenter.py

```diff
@@ -82,6 +82,9 @@ def _get_workspace_list(self):
         fit_context = self.model._fit_context
         workspace_list = []
         fit_list = fit_context.fit_list
+        if len(fit_list) == 0:
```

Comment:
Would `if not fit_list:` be more pythonic?

Reply:
Potentially yes. I like checking the length because I think it is clearer what I am checking for and why the code is there, but relying on the falseness of an empty list is probably more pythonic.

Reply:
Ok, I am happy for it to stay as it is